packages/lesmis-server/src/connections/EntityConnection.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+import camelcaseKeys from 'camelcase-keys';
+import { ApiConnection } from '@tupaia/server-boilerplate';
+import { LESMIS_PROJECT_NAME, LESMIS_COUNTRY_ENTITY_CODE } from '../constants';
+
+const { ENTITY_SERVER_API_URL = 'http://localhost:8050/v1' } = process.env;
+
+export class EntityConnection extends ApiConnection {
+  baseUrl = ENTITY_SERVER_API_URL;
+
+  async getEntities() {
+    const response = await this.get(
+      `hierarchy/${LESMIS_PROJECT_NAME}/${LESMIS_COUNTRY_ENTITY_CODE}/descendants`,
```

Comment:
```suggestion
      `hierarchy/${LESMIS_PROJECT_NAME}/${LESMIS_COUNTRY_ENTITY_CODE}/descendants?includeRootEntity=true`,
```
New addition in Rohan's PR. Needed here or it won't include the "Laos" country entity itself. Alternative would be to pass the lesmis project name as the entity as well, as then it will be the root entity and return everything underneath (which would include the country)

Reply:
cool. I have a slight preference for the project as the entity as it fits well conceptually for me. ie get all the entities for the lesmis project